cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -187,16 +187,28 @@ def create_xblock(parent_locator, user, category, display_name, boilerplate=None
         metadata = {}
         data = None
         template_id = boilerplate
-        if template_id:
-            clz = parent.runtime.load_block_type(category)
-            if clz is not None:
-                template = clz.get_template(template_id)
+        default_display_name = None
+        cls = parent.runtime.load_block_type(category)
```

Comment:
`cls` is a reserved Python keyword, like `self`, so you shouldn't use it as a variable name. What is this - a block category? then `block_category` would be better. Be descriptive with the variable name.